4b/Grains/src/Test1.cs

```diff
@@ -1,14 +1,38 @@
+using System;
 using System.Threading.Tasks;
 using Interfaces;
+using Interfaces.TRX;
 
 namespace Grains
 {
     public class Test1 : Orleans.Grain, ITest1
     {
-        public async Task<bool> HelloWorldTest()
+        public async Task<UnitTest> HelloWorldTest()
         {
             await Task.Delay(1000);
```

Comment:
Let's reduce this by 10x.
```suggestion
            await Task.Delay(1000);
```
```suggestion
            await Task.Delay(100);
```